transformations/concat/README.md

```diff
@@ -0,0 +1,31 @@
+# Concatenate Two Random Sentences
+This transformation concats two random sentences as explained in [Nguyen et al., 2021](https://arxiv.org/abs/2105.01691). This is the bilingual version. A monolingual version is available under concat monlingual.
+
```

Comment:
Could you please add your information here, name/contact/institution, etc.?

Reply:
Done. Thanks!